src/components/cards/cards.css

```diff
@@ -0,0 +1,125 @@
+.card
+{
+    border-radius: var(--rounded-md);
+    background-color: var(--light);
+    padding: var(--u-10);
+    transition: var(--normal-transition);
+}
+
+.card:hover
+{
+    box-shadow: none;
+}
+
+.card-vert
+{
+    width: 15.5rem;
+}
+
+
+.card-header
+{
+    position: absolute;
```

Comment:
instead of using position absolute, you can use grid overlay which will be better maintainable not that there is something wrong with this.